### ISSUE TITLE
list index out of range on mylfblows scene

### ISSUE THREAD

**Issue by d2dyno** [PAhelper/PhoenixAdult.bundle#245]
Trying to match using `mylf 1837/after-school-milf-suck`, and logs produce:

```
2019-07-03 01:57:29,663 (14923291e700) :  INFO (agentkit:957) - Searching for matches for {'year': None, 'id': '103515', 'name': 'Mylf 1837/after-school-milf-suck'}
2019-07-03 01:57:29,664 (14923291e700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/103515/tree'
2019-07-03 01:57:29,669 (14923291e700) :  INFO (__init__:46) - *******MEDIA TITLE****** Mylf 1837/after-school-milf-suck
2019-07-03 01:57:29,669 (14923291e700) :  INFO (__init__:53) - Getting Search Settings for: Mylf 1837/after-school-milf-suck
2019-07-03 01:57:29,669 (14923291e700) :  INFO (PAsearchSites:880) - mediaTitle w/ possible abbrieviation: Mylf 1837/after school milf suck
2019-07-03 01:57:29,677 (14923291e700) :  INFO (PAsearchSites:1018) - mediaTitle w/ possible abbrieviation fixed: Mylf 1837/after school milf suck
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1033) - ^^^^^^^ siteID: 674
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1035) - ^^^^^^^ Shortening Title
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1036) - mylf vs mylf
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1039) - 1
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1088) - 6
2019-07-03 01:57:29,680 (14923291e700) :  INFO (PAsearchSites:1088) - searchTitle (before date processing): 1837/after school milf suck
2019-07-03 01:57:29,680 (14923291e700) :  INFO (__init__:63) - Search Title: 1837/after school milf suck
2019-07-03 01:57:29,681 (14923291e700) :  INFO (__init__:72) - 1837/after%20school%20milf%20suck
2019-07-03 01:57:29,683 (14923291e700) :  INFO (siteMylf:9) - searchString: 1837/after-school-milf-suck
2019-07-03 01:57:29,685 (14923291e700) :  DEBUG (networking:161) - Fetching 'https://www.mylf.com/movies/1837/after-school-milf-suck' from the HTTP cache
2019-07-03 01:57:29,689 (14923291e700) :  CRITICAL (agentkit:1014) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '103515', 'name': 'Mylf 1837/after-school-milf-suck'} (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-158e5b199/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1007, in _search
    agent.search(*f_args, **f_kwargs)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 769, in search
    results = PAsearchSites.siteMylf.search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate, searchSiteID)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/siteMylf.py", line 15, in search
    titleNoFormatting = searchResults.xpath('//div[@class="col-12 col-md-7"]//span[contains(@class,"text-lightgray")]')[0].text_content().strip()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-158e5b199/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range
```

**Comment by kamuk90** [comment]
replace line 15 in siteMylf.py
```
    titleNoFormatting = searchResults.xpath('//div[@class="col-12 col-md-8"]//span[contains(@class,"text-lightgray")]')[0].text_content().strip()
```

**Comment by fritz276** [comment]
Thanks, that helped for the mylf sites. No posters, though.

**Comment by claygoldfinch** [comment]
Resolved in PR #264
